apps/reference/docs/guides/platform/logs.mdx

```diff
@@ -4,78 +4,177 @@ title: Logging
+The features discussed in this article are only available through the **Supabase Platform** and are not yet available on self-hosted.
 
 :::
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ## Product Logs
 
-As well as a Log Explorer, Supabase provides a logging interface specific to each product.
+Supabase provides a logging interface specific to each product.
 
-### API Logs
+Simple regex searching is available through the search bar to search the log event messsage for specific keywords and patterns.
 
-![API Logs](/img/guides/platform/logs/logs-api.png)
+You may also export and download the logs as a spreadsheet.
+
+<!-- 
+To update the screenshots, ensure that at least one log line is selected to display the metadata. Can use meme.town as an exmaple.
+-->
+
+<Tabs>
+  <TabItem value="api" label="API" default>
 
 The API Logs can be found under `Database > API Logs`. These show all network requests for the REST and GraphQL [API](../../guides/api).
 
-### Postgres Logs
+![API Logs](/img/guides/platform/logs/logs-api.png)
 
-![Postgres Logs](/img/guides/platform/logs/logs-database.png)
+  </TabItem>
+  <TabItem value="postgres" label="Postgres">
 
 The Postgres Logs can be found under `Database > Postgres Logs`. These show all queries and activity for your [Database](../../guides/database).
 
-## Log Explorer
+![Postgres Logs](/img/guides/platform/logs/logs-database.png)
+
+  </TabItem>
+  <TabItem value="auth" label="Auth">
+
+The Auth Logs can be found under `Authentication > Logs`. These show all server logs for your [Auth system](../../guides/auth).
+
+![Auth Logs](/img/guides/platform/logs/logs-auth.png)
+
+  </TabItem>
+  <TabItem value="storage" label="Storage">
+
+The Storage Logs can be found under `Storage > Logs`. These show all server logs for your [Storage API](../../guides/storage).
+
+![Storage Logs](/img/guides/platform/logs/logs-storage.png)
+
+  </TabItem>
+  <TabItem value="realtime" label="Realtime">
+
+The Realtime Logs can be found under `Database > Realtime Logs`. These show all server logs for your [Realtime API](../../guides/realtime).
 
-![Sidebar navigation steps](/img/guides/platform/logs/sidebar-navigation.png)
+![Realtime Logs](/img/guides/platform/logs/logs-realtime.png)
+
+  </TabItem>
+  <TabItem value="functions" label="Edge Functions">
+
+The Edge Functions Logs can be found under two tabs:
+- `Functions > Your Function > Invocations`: Edge logs of function calls.
+- `Functions > Your Function > Logs`: Logs emitted during function execution.
+
+![Function Edge Logs](/img/guides/platform/logs/logs-functions-edge.png)
+
+![Function Logs](/img/guides/platform/logs/logs-functions.png)
+
+  </TabItem>
+</Tabs>
+
+## Logs Explorer
+
+![Logs Explorer](/img/guides/platform/logs/logs-explorer.png)
 
 The log browser can be accessed in the sidebar under **Logs Explorer**. The **Logs Explorer** is for querying and aggregating project logs across products using SQL `SELECT` queries.
```

Comment:
```suggestion
The Dashboard includes a **Logs Explorer** for querying and aggregating logs across products using SQL `SELECT` queries. This can be accessed in the sidebar under **Logs Explorer**. 
```